LayoutTests/platform/ios/TestExpectations

```diff
@@ -3743,6 +3743,10 @@ imported/w3c/web-platform-tests/css/css-writing-modes/forms/color-input-appearan
 imported/w3c/web-platform-tests/css/css-writing-modes/forms/color-input-appearance-none-horizontal.optional.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-writing-modes/forms/color-input-appearance-none-vertical.optional.html [ ImageOnlyFailure ]
 
+# <select> are always menulists on iOS, regardless of multiple/size attributes, these tests assume the listbox appearance that desktop platforms have.
```

Comment:
Is it really OK that WPT requires this?

Reply:
I added this test some we have some testing coverage for vertical form controls for Interop. It has the `.optional.html` suffix (like most of these tests do) for this reason. I think it's desirable for all desktop browsers (and even some mobile browsers) to have this test, so this is why I put it there. It can always be moved afterwards if necessary.